test/unit/org/apache/cassandra/index/accord/RouteIndexTest.java

```diff
@@ -130,14 +130,16 @@ public static void setUpClass()
         // this flag disables that flush
         CassandraRelevantProperties.UNSAFE_SYSTEM.setBoolean(true);
 
+        setupFileSystem();
```

Comment:
this needs to happen before `DatabaseDescriptor.daemonInitialization();` else `auto` gets resolved to `direct`.  We need to process `auto` on the file system the test is going to run against